mmv1/third_party/terraform/provider/provider_mmv1_resources.go.erb

```diff
@@ -47,6 +47,7 @@ var handwrittenDatasources = map[string]*schema.Resource{
 	"google_cloud_identity_groups":                     cloudidentity.DataSourceGoogleCloudIdentityGroups(),
 	"google_cloud_identity_group_memberships":          cloudidentity.DataSourceGoogleCloudIdentityGroupMemberships(),
 	"google_cloud_identity_group_lookup":               cloudidentity.DataSourceGoogleCloudIdentityGroupLookup(),
+	"google_cloud_quotas_quota_info":                   cloudquotas.DataSourceGoogleCloudQuotasQuotaInfo(),
```

Comment:
I think we need to manually import the package- it's normally automatic, but this package has no resources and the automation's probably triggered off of including at least one.

Reply:
nvm- this does work

Reply:
Oh wait, it works in beta but not GA. Weird, I don't know why. Trigger is almost certainly that there are no resources. We can add a version guard for only GA (`if version == ga`), I guess, since other levels seem to work properly.

Reply:
sg, I'm seeing other services using `unless version == 'ga'`, does `if` and `unless` work the same way? Also if I add additional resources for the service later on, will I be able to remove the import?

Reply:
`unless expr` is `if !(expr)`, so `unless version == 'ga'`  is `if !(version == GA)`. Those mark the minimum version of a line as beta. Since beta's working fine here and GA isn't we're using `if` instead of `unless`, marking the _maximum_ instead.

> Also if I add additional resources for the service later on, will I be able to remove the import?

Yeah, it should get generated automatically when a resource is added and we'll need to remove it then. This is just patching for a weird glitch we ran into. Tracking bug is https://github.com/hashicorp/terraform-provider-google/issues/15633 where I've commented to make sure we track somewhere we've patched it, in case we clean it up by fixing whatever the issue here is.